lib/src/view/design_side_helix.dart

```diff
@@ -92,6 +94,14 @@ class DesignSideHelixComponent extends UiComponent2<DesignSideHelixProps> with P
               ? temp_text_for_help_doc_figure_making
               : props.helix.idx.toString(),
           Dom.svgTitle()(tooltip)),
+      (Dom.text()
```

Comment:
Instead of always rendering the text element, and giving it an empty string sometimes, it's cleaner simply not to render it at all if `props.show_grid_coordinates` is false. Replace this with 

```dart
if (props.show_grid_coordinates)
  (Dom.text() 
  // ...
```

That's the [collection if](https://dart.dev/guides/language/language-tour#collection-operators) operator. (explained more here: https://medium.com/dartlang/announcing-dart-2-3-optimized-for-building-user-interfaces-e84919ca1dff)